google_secrets_manager_client/encryption.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.kdf.scrypt import Scrypt
+from google_secrets_manager_client.secrets_manager import access_secret_version
+
+salt = b"LOM4h0lOzLG1MDnL"
+iv = b"iV2Jw7WufiFdxoUx"
```

Comment:
this is better be kept with the encrypted string and generated randomly, like password hashing and it's salt